utils/get_licenses_from_ort.py

```diff
@@ -34,8 +35,13 @@
     "Zlib",
     "MIT OR Unlicense",
     "PSF-2.0",
+    "Unicode-3.0",
+    "(Apache-2.0 OR MIT) AND Unicode-3.0"
+]
+APPROVED_PACKAGES = [
```

Comment:
Please add a description, such as “Packages with non-pre-approved licenses that received manual approval.”